app/models/api/transliterate.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+// Check that the class exists before trying to use it
+if (! class_exists('Transliterator')) {
+    $request->error = 'Class Transliterator not present';
+    $json = $request->invalidAPICall();
+
+    return;
+}
+
+switch ($request->parameters[2]) {
+    case 'serbian':
```

Comment:
We shouldn't use locale names but locale codes. In this case, the locale code would be sr-Cyrl